### DISCUSSION TITLE
pinch/zoom?

### DISCUSSION THREAD

**Discussion by mgbnut** [AppGround-io/bluetooth-keyboard-and-mouse-support#122] [Ideas]
I tried a few different mouse/keyboards and this works beautifully. However, I expected pinch/zoom to work. it doesn't, of course, am I missing something?

**Comment by Frambooisier** [comment]
it does work for me on Motorola one zoom, maybe try to make slower but wider swipes.

**Reply by Frambooisier** [reply]
also, it would be useful to know on what devices are you testing the app. It could be that the specific device is not compatible with that function.
